src/Umbraco.Core/Install/Models/DatabaseModel.cs

```diff
@@ -12,16 +12,16 @@ public class DatabaseModel
     public string? ProviderName { get; set; }
 
     [DataMember(Name = "server")]
-    public string Server { get; set; } = null!;
+    public string? Server { get; set; }
```

Comment:
I reckon making these nullable counts as a breaking change? If that's intended just ignore this comment 😄

Reply:
I reckon you're quite right 😄, didn't forgot about breaking changes for a second 🤦, I've made this non-nullable again and added a todo to re-nullify it, since the Server part can be nullable, for instance when installing SQLite, and we even have checks for it :D